### ISSUE TITLE
Bug: v8.0.0 + arrow functions + JSX breaks some highlighting

### ISSUE THREAD

**Issue by hawkrives** [babel/babel-sublime#162]
<img width="649" alt="screen shot 2015-08-19 at 4 36 50 pm" src="https://cloud.githubusercontent.com/assets/464441/9369692/888c8854-4690-11e5-8be0-3ffb598459f6.png">

```jsx
<article className={styles['completed-work']}>
	<div>
		{props.secondary.map(pic => 
			<img src={pic} />)}
	</div>
	<div>
		{props.secondary.map(pic => { return <img src={pic} /> })}
	</div>
	<div>
		{props.secondary.map(pic => <img src={pic} />)}
	</div>
</article>
```

The arrow function in the last `<div>` throws off the syntax highlighting for everything that follows it. It only seems to happen if a JSX element is implicitly returned from the same line as an arrow expression.

Let me know if there's anything else you need!

And thanks for all your dedication to this project!

**Comment by hawkrives** [comment]
- babel-sublime@8.0.0
- Sublime Text build 3095 (dev channel)
- JavaScript (Babel) syntax

**Comment by zertosh** [comment]
Thanks for the report! I broke it last night in https://github.com/babel/babel-sublime/pull/160, but it's now fixed by https://github.com/babel/babel-sublime/commit/6ce10a2f0c0fbdf8827e66456118c644ac2c8a61. Just published it as v8.0.1 – it'll be in Package Control in a few minutes.

**Comment by hawkrives** [comment]
Thanks!
